src/host/outputStream.hpp

```diff
@@ -45,7 +45,7 @@ class ConhostInternalGetSet final : public Microsoft::Console::VirtualTerminal::
 
     void SetWindowTitle(const std::wstring_view title) override;
 
-    void UseAlternateScreenBuffer() override;
+    void UseAlternateScreenBuffer(const TextAttribute& attrs) override;

```

Comment:
What if we made this `const TextAttribute& attrs = {}`? That way we wouldn't have to touch the existing code, and I feel like it would still be just as readable despite the implicit overload.

Reply:
I considered that, but I thought it would give the wrong impression, because the default behavior for the alt buffer should not be to fill with an empty `TextAttribute` - that's really the opposite of the default. It's convenient in the unit tests because it's easy to construct, and those tests weren't reliant on it being a particular value, but it's not correct in general.

And long term I'm hoping we'll be able to get rid of these APIs anyway. We really shouldn't have ConHost and Terminal having entirely separate implementations for the alt buffer.